site-config/hca-dcp/cc-ma-dev/index/filesEntityConfig.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  ComponentConfig,
+  EntityConfig,
+} from "@databiosphere/findable-ui/lib/config/entities";
+import * as C from "../../../../app/components";
+import * as V from "../../../../app/viewModelBuilders/azul/hca-dcp/common/viewModelBuilders";
+import { COLUMN } from "./common/constants";
+
+/**
+ * Returns the entity config for managed access files.
+ * @param entitiesConfig - Entities config.
+ * @returns entity config for managed access files.
+ */
+export function getMAFilesEntityConfig(
+  entitiesConfig: EntityConfig
+): EntityConfig {
+  // Clone files entity.
+  const cloneEntity = { ...entitiesConfig };
+  // Clone list.
+  const cloneList = { ...cloneEntity.list };
+  // Clone columns.
+  const cloneColumns = [...cloneList.columns];
+  // Add data use restriction column.
+  cloneColumns.splice(2, 0, {
+    ...COLUMN.DATA_USE_RESTRICTION,
+    componentConfig: {
+      component: C.BasicCell,
+      viewBuilder: V.buildAggregatedDataUseRestriction,
+    } as ComponentConfig<typeof C.BasicCell>,
```

Comment:
@jpaten, if we should be displaying all possible `dataUseRestriction` values on the `filesResponse`, let's change the component on lines 27 and 29 to `C.NTagCell`.